doc/source/whatsnew/v0.21.0.txt

```diff
@@ -46,12 +46,11 @@ Backwards incompatible API changes
 
 - Support has been dropped for Python 3.4 (:issue:`15251`)
 - The Categorical constructor no longer accepts a scalar for the ``categories`` keyword. (:issue:`16022`)
-
 - Accessing a non-existent attribute on a closed :class:`HDFStore` will now
   raise an ``AttributeError`` rather than a ``ClosedFileError`` (:issue:`16301`)
 - :func:`read_csv` now treats ``'null'`` strings as missing values by default (:issue:`16471`)
 - :func:`read_csv` now treats ``'n/a'`` strings as missing values by default (:issue:`16078`)
-
+- Compression defaults in HDF stores now follow pytable standards. Default is no compression and if complib is missing and complevel > 0 zlib is used (:issue:`15943`)
```

Comment:
use double ticks to highlite thing, e.g. (keywords args)

``zlib``, ``complib`` and ``complevel > 0``